recipe/multitenancy/multitenancyClaims.go

```diff
@@ -0,0 +1,84 @@
+package multitenancy
+
+import (
+	"errors"
+	"time"
+
+	"github.com/supertokens/supertokens-golang/recipe/multitenancy/multitenancyclaims"
+	"github.com/supertokens/supertokens-golang/recipe/session/claims"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func NewMultitenancyClaims() (*claims.TypeSessionClaim, *claims.TypeSessionClaim, multitenancyclaims.TypeMultitenancyClaimValidators) {
+	fetchTenantId := func(userId string, userContext supertokens.UserContext) (interface{}, error) {
+		instance, err := GetRecipeInstanceOrThrowError()
+		if err != nil {
+			return nil, err
+		}
+		tenantIdRes, err := instance.GetTenantIdForUserID(userId, userContext)
+		if err != nil {
+			return false, err
+		}
+		if tenantIdRes.OK != nil {
+			return tenantIdRes.OK.TenantId, nil
```

Comment:
if the user has no tenantId, or has default tenantId, we do not want to add this claim to the session at all. Idk if that's possible to do effeciently cause this means each time a multi tenancy validator is used, the sdk will call fetch value in case the claim doesn't exist (as opposed to treating it as if the user has no tenant ID).. to discuss.

Reply:
this claim is removed